LivingCity/traffic/b18TrafficPerson.h

```diff
@@ -113,11 +125,18 @@ struct B18TrafficVehicle {
                isInIntersection == other.isInIntersection;
     }
 };
+
 struct B18TrafficVehicleModify{
   bool ifToCopy;
   bool ifToRemove;
   int gpuIndexToCopy;
 };
+
+struct B18TrafficPerson{
+  int id;
+  Node* transferPoints; //should take which bus line and drop at which intersection
```

Comment:
Add num_step to this property to track travel time.
To be fix: init the first segment of trip in transfer point to save travel time